pages/common/pulumi.md

```diff
@@ -0,0 +1,28 @@
+# pulumi
+
+> Define infrastructure on any cloud using familiar programming languages.
+> More information: <https://www.pulumi.com/docs/>.
+
+- Create a new project using a template:
+
+`pulumi new`
+
+- Create a new stack, an isolated deployment target:
+
+`pulumi stack init`
+
+- Configure variables such as keys, regions, and so on:
```

Comment:
Does this open a interactive command or is there other parameters that need to go with it?

Reply:
It opens an Interactive command  but you can also provide additional parameters.

Reply:
such as?

Reply:
Such as an Organization name or/and a Stack Name

Reply:
Do you input them once when the command is running or as parameters in the command?

Reply:
You input them when the command is running

Reply:
Ah, ok thanks.


```suggestion
- Interactively configure variables such as keys, regions, etc: 
```